code/game/objects/items/weapons/legcuffs.dm

```diff
@@ -41,7 +41,11 @@
 
 /obj/item/restraints/legcuffs/beartrap/attack_self(mob/user)
 	..()
-	if(ishuman(user) && !user.stat && !user.restrained())
+
+	if(!ishuman(user) || ishuman(user) && user.restrained())
+		return
+
+	if(do_after(user, 2 SECONDS, target = user))
```

Comment:
```suggestion
	if(do_after(user, 2 SECONDS))
```
My bad this also doesn't work. Was there any goal you tried with the target being on the user?

Reply:
Talked about this over discord with each other and resolved the issue.